tests/ExtendsFinalClassTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Psalm\Tests;
+
+class ExtendsFinalClassTest extends TestCase
+{
+    use Traits\InvalidCodeAnalysisTestTrait;
+
+    /**
+     * @return iterable<string,array{string,error_message:string,2?:string[],3?:bool,4?:string}>
+     */
+    public function providerInvalidCodeParse()
```

Comment:
I'd suggest adding a couple of tests showing how this new issue could be suppressed (with a docblock on the extending class, I suppose?).

Reply:
Thanks for the suggestions. Added two examples for suppression with docblock.